Documentation/gettingstarted/cni-chaining-azure-cni.rst

```diff
@@ -39,7 +41,7 @@ example below, the Azure CNI, portmap, and Cilium are chained together.
     kind: ConfigMap
     metadata:
       name: cni-configuration
-      namespace: cilium
+      namespace: kube-system
```

Comment:
This looks a little incidental, why was `cilium` namespace used previously? If this has to be changed in this PR, a separate commit would be nice.

Reply:
I proposed this change earlier, but someone (@aanm?) mentioned there used to be a service (on GKE?) that periodically removed certain resources from `kube-system`. Not sure if this is still the case.

Reply:
yes, that was the case on GKE, I'm not sure about AKS.

Reply:
Alright, should be fine then :+1: